src/app/docs/dropdown/dropdown-docs.component.html

```diff
@@ -0,0 +1,330 @@
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <sky-docs-design-guideline
+        heading="Dropdown menu action text"
+      >
+        <p>
+          For an action that clearly applies to the object associated with the context menu, use the following format for the action wording:
+        </p>
+        <p>
+          <span class="sky-emphasized">&lt;Verb&gt;</span>
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/wording/context-menu.png"
+          showBorder="true"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline>
+          <p>
+            For a dropdown button with one type of action, use the following format:
+          </p>
+          <p>
+            Button text: <span class="sky-emphasized">&lt;Verb&gt;</span>
+          </p>
+          <p>
+            Menu item: <span class="sky-emphasized">&lt;Direct object&gt;</span>
+          </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/wording/dropdown-single-action.png"
+          showBorder="true"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline>
+          <p>Additional button text guidelines:</p>
+          <ul>
+            <li>Use sentence-style capitalization.</li>
+            <li>Make the direct object singular or plural depending on whether the action applies to one or multiple items.</li>
+            <li>Don’t append extra words such as “details.” For example, use “Edit constituent” rather than “Edit constituent details.”</li>
+          </ul>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Layout"
+    >
+      <sky-docs-design-guideline>
+          <p>Place context menu dropdowns to the left of all row-specific content but to the right of all list controls such as expand-collapse actions in tree views and checkboxes in grids or repeaters.</p>
+          <p>Default dropdowns are positioned with the same alignment and margins as SKY UX buttons.</p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/layout.png"
+          captionType="success"
+          caption="put context menu dropdowns to the left of all row-specific content in lists."
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Related information"
+    >
+      <sky-docs-design-guideline
+       heading="Components"
+      >
+        <ul>
+          <li><a routerLink="/components/button">Buttons</a></li>
+          <li><a routerLink="/components/popover">Popovers</a></li>
```

Comment:
These routerLinks will need to be converted to a hard URL (`href`) pointing to the page on the main docs site.